pkg/core/runtime/builder.go

```
@@ -70,12 +72,14 @@ type Builder struct {
 }
 
 func BuilderFor(cfg kuma_cp.Config) *Builder {
+	hostname, _ := os.Hostname()
```

please do not ignore errors. Change the signature to
```
func BuilderFor(cfg kuma_cp.Config) (*Builder, error)
```